ui/src/app/constants/texts.constants.ts

```diff
@@ -51,6 +51,12 @@ export const texts = {
   RUN_WORKFLOWS_SUCCESS_NOTIFICATION: 'Workflows have been submitted.',
   RUN_WORKFLOWS_FAILURE_NOTIFICATION: "Sorry, workflows couldn't be submitted. Please try again.",
 
+  KILL_JOB_CONFIRMATION_TITLE: 'Kill running job',
+  KILL_JOB_CONFIRMATION_CONTENT:
+    'Are you sure you want to kill running job? The operation cannot be reverted and operation may not take effect.',
```

Comment:
This is correct, but it sounds a bit confusing. I would only keep "The operation cannot be reverted" and omit "operation may not take effect"

Reply:
Ok